.claude/skills/icon-download/SKILL.md

```
@@ -0,0 +1,21 @@
+---
+name: icon
+description: 必要なアイコンの説明からGoogle Fontsで適切なアイコンを探し、Android Vector DrawableのXMLをres/drawableにダウンロードする
+argument-hint: <アイコンの説明>
+allowed-tools: Bash(curl *) Write WebFetch
+---
+
+$ARGUMENTS に適した検索ワードを考え、以下のURLで検索してアイコン名を特定する：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)